IdentityCore/src/throttling/model/MSIDThrottlingModelFactory.m

```diff
@@ -40,7 +40,8 @@ + (MSIDThrottlingModelBase *)throttlingModelForIncomingRequest:(id<MSIDThumbprin
     if (![MSIDThrottlingModelFactory validateInput:request]) return nil;
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->